world/console.cpp

```diff
@@ -1195,11 +1263,27 @@ void ConsoleWorldWideMove(
 		connection->SendLine("wwmove [zone_short_name] [min_status] [max_status]");
 		return;
 	}
+\
+	const auto&  zone_short_name = !Strings::IsNumber(args[2]) ? args[2] : "";
+	const uint16 instance_id     = Strings::IsNumber(args[2]) ? static_cast<uint16>(Strings::ToUnsignedInt(args[2])) : 0;
 
-	const auto&  zone_short_name  = !Strings::IsNumber(args[0]) ? args[0] : "";
-	const uint16 instance_id      = Strings::IsNumber(args[0]) ? static_cast<uint16>(Strings::ToUnsignedInt(args[0])) : 0;
-	const uint8  min_status       = args.size() >= 2 ? static_cast<uint8>(Strings::ToUnsignedInt(args[1])) : 0;
-	const uint8  max_status       = args.size() >= 3 ? static_cast<uint8>(Strings::ToUnsignedInt(args[2])) : UINT8_MAX;
+	const auto& z = !zone_short_name.empty() ? zone_store.GetZone(zone_short_name) : nullptr;
+
+	if (!zone_short_name.empty() && !z->id) {
```

Comment:
Might want to just do `if (z && !z->id)` because `z` could be null and while `zone_short_name` might not be empty, it might not actually return a valid pointer and return `nullptr`